arch/arm/core/cortex_m/mpu/arm_core_mpu.c

```diff
@@ -52,4 +52,37 @@ int _arch_mem_domain_max_partitions_get(void)
 {
 	return arm_core_mpu_get_max_domain_partition_regions();
 }
+
+/**
+ * @brief Reset MPU region for a single memory partition
```

Comment:
you don't need to put documentation here.
document this in the arch-independent header